ckanext/subscribe/email_auth.py

```diff
@@ -46,44 +66,30 @@ def send_subscription_confirmation_email(code, subscription=None):
                           headers={})
 
 
-def get_subscription_confirmation_email_contents(code, subscription):
-    email_vars = get_email_vars(subscription=subscription, code=code)
-    plain_text_footer, html_footer = \
-        get_footer_contents(code, subscription=subscription)
+def send_manage_email(code, subscription=None, email=None):
```

Comment:
what do we return for this function? None?

Reply:
Yep, `None`.